Sources/RunCatLocalization/Resources/RunnersStore.xcstrings

```diff
@@ -485,6 +587,12 @@
             "value" : "Reload"
           }
         },
+        "fr" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Actualiser"
```

Comment:
Based on the wording used in Apple's default apps, wouldn't `Recharger` be more appropriate?
https://applelocalization.com/macos?q=Reload&b=Safari.app&l=English&l=French